adds feature blocks

leejordan opened the pull request:
I've standardised some of the feature blocks used on the shop and improved their flexibility in terms of layout options and art direction for photography.

![pistachio-features](https://cloud.githubusercontent.com/assets/713293/16299318/8d1788ee-3931-11e6-95e0-bca05f484991.png)

notlee commented on less/modules/features.less:
```diff
@@ -0,0 +1,142 @@
+    width: 100%;
+    margin-top: auto;
+    margin-bottom: 0;
+    padding: 0.5em 1em;
```
Should we be sticking to variables to define paddings/margins consistently?

notlee commented on less/modules/features.less:
```diff
@@ -0,0 +1,142 @@
+        background-size: 150%;
+        background-position: top right;
+
+        .feature__side {
```
This classname doesn't feel very intuitive. Would something like `.feature__content` make more sense?

notlee commented:
@leejordan Awesome! 💃 

Found an issue with text not wrapping IE10:
![screen shot 2016-06-24 at 11 59 19](https://cloud.githubusercontent.com/assets/10405691/16336298/3116d8f0-3a04-11e6-96a0-db7875c4d738.png)

And there might be an annoying rendering issue in IOS potentially which may or may not be worth looking at:
![screen shot 2016-06-24 at 12 05 56](https://cloud.githubusercontent.com/assets/10405691/16336310/47c042e4-3a04-11e6-9c39-27e276d9c894.png)

leejordan commented:
@notlee I have addressed your comments apart form the ios bug which is proving tricky to fix so I'll leave it for another day as it's not a breaking bug
